operator/deploy/crd.yaml

```diff
@@ -30,11 +30,335 @@ spec:
     listKind: KialiList
     plural: kialis
     singular: kiali
+  preserveUnknownFields: false
   scope: Namespaced
-  subresources:
-    status: {}
-  version: v1alpha1
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
+    - name: v1alpha1
```

Comment:
The array indentation in here looks wrong. All of our other yaml indentations (at least the yaml in the operator) has arrays on the same indentation level as the parent. So:

```yaml
versions:
- name
```

not

```yaml
versions:
  - name
```

Reply:
I used the indentation as in this sample: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition

I'm not sure if there is any difference.

Reply:
There isn't a difference so long as a yaml file uses one way or the other.

But all the operator yaml files use indentation of lists on the same level as the parent item. For example, in the current CRD, you will see this:

https://github.com/kiali/kiali/blob/v1.9/operator/deploy/crd.yaml#L38

```yaml
  versions:
  - name: v1alpha1
    served: true
    storage: true
```

And this is how the indentation is everywhere else. So we want to keep this file consistent and use the same indentation as every other yaml file.

Reply:
Right, I'll fix all instances.